app/views/News.js

```diff
@@ -127,16 +100,38 @@ class NewsScreen extends Component {
               style={{
                 backgroundColor: Colors.VIOLET_BUTTON_DARK,
                 flex: 1,
-                paddingVertical: 16,
+                flexDirection: 'row',
+                alignItems: 'center',
+                justifyContent: 'center',
+                paddingHorizontal: 16,
               }}>
-              <Carousel
-                data={this.state.newsUrls}
-                renderItem={this.renderItem}
-                sliderWidth={width}
-                itemWidth={width * 0.85}
-                layout={'default'}
-                scrollEnabled
-              />
+              <ScrollView style={{ width: width * 0.85 }}>
+                {this.state.newsUrls.map((item, index) => (
+                  <View key={index} style={styles.singleNews}>
+                    <View key={index} style={styles.singleNewsHead}>
+                      <Typography style={styles.singleNewsHeadText}>
+                        {item.name}
+                      </Typography>
+                    </View>
+                    <WebView
+                      source={{
+                        uri: item.news_url,
+                      }}
+                      style={{ height: height * 0.75 }}
+                      containerStyle={{
+                        borderBottomLeftRadius: 12,
+                        borderBottomRightRadius: 12,
+                      }}
+                      cacheEnabled
+                      onLoad={() =>
+                        this.setState({
+                          visible: false,
```

Comment:
small nitpick - could you change the state value to `isVisible` to better indicate its a boolean